woocommerce/woocommerce-ios
# Coupons: Add feedback banner on top of coupon list

Closes: #6288 


### Description

This PR adds a feedback banner on the top of the coupon list to gather user feedback for the experimental feature.
Changes include:
- Storage: add coupon management as a new feedback type
- Yosemite: update in-app feedback card visibility use case with the new feedback type
- UI: Update coupon list and view model to show/hide the feedback banner based on view state and feedback banner visibility setting
- Other: Add and save URLs for coupon management survey on Crowd Signal.

### Testing instructions

- Make sure that your test store has enabled coupons (WP Admin > WooCommerce > Settings > General > Enable Coupons).
- Make sure that your test store has at least one published coupons (WP Admin > Marketing > Coupons)
- On the app, turn on Coupon Management in Menu > Settings > Experimental Features.
- Navigate to Menu > Coupons. After the list is loaded successfully, notice the feedback banner at the top. This banner can be toggled to expand or collapse; and can be dismissed entirely for that session if you tap on the (x) button.

### Screenshots

<img src="https://user-images.githubusercontent.com/5533851/155922144-1983a122-238e-4d12-9d04-b5a5a6d8a127.png" width=320 />

---
